src/earn/poolInfoScreen/EarnPoolInfoScreen.test.tsx

```diff
@@ -557,34 +553,6 @@ describe('EarnPoolInfoScreen', () => {
     expect(getByTestId('Earn/ActionCard/Transfer')).toBeTruthy()
   })
 
-  it('navigates correctly when swap action item is tapped', () => {
```

Comment:
swap action is still possible when the pool doesn't support swap and deposits (e.g., all bridge). Should we retain this test or is there another test that covers this scenario?